README.md

```diff
@@ -8,11 +8,21 @@ Initial work in progress to create a python library for SPIFFE.
 The _RFC - Py-Spiffe features and use cases_ document is public at: https://docs.google.com/document/d/1IByE9Ge1MTyWD7rL9o99-1c5Fn83p1ra3eApARY9X-I/edit#heading=h.70d940e8vk81
 
 ## Initial contributors
```

Comment:
super nit picky, but wondering if we should standarize on "FirstName LastName", my OCD isn't letting me see this 😂 
@Andres-GC @maxlambrecht would you mind?

Reply:
+1 to list all names in the same format. 
I don't have any preference between Name-Lastname or Lastname-Name

Reply:
Thanks folks! Done!